File: test12/perl_total_bill/total_bill.pl
#!/usr/bin/perl -w


$total = 0;

while (<>) {
    if (m|("price": "\$)(\d+.\d+)|) {
        $total += $2;
    }
    # print;
    # print "$2\n";
}

printf("\$%.2f\n", $total);

File: test12/js_total_bill/total_bill.js
function total_bill(bill_list) {
  let total = 0;
  for (const bill of bill_list) {
    for (const item of bill) {
      total = total + Number(item.price.slice(1));
    }
  }
  return total;

}

module.exports = total_bill;
